.travis.yml

```diff
@@ -34,6 +34,10 @@ env:
   - VIBED_DRIVER=libasync BUILD_EXAMPLE=0 RUN_TEST=0
 
 matrix:
+  include:
+    - d: dmd
+      # https://issues.dlang.org/show_bug.cgi?id=13742
+      env: DUB_ARGS=--build=unittest-cov --build-mode=singleFile
```

Comment:
Not very familar with YAML, but form the error message it seems like this may need some form of quotes.